larray_editor/utils.py

```diff
@@ -48,36 +50,24 @@ def get_versions():
         'qt': qtpy_version,
         'qt_api': API_NAME,                             # PyQt5 or PyQt4
         'qt_api_ver': PYQT_VERSION,
+        'matplotlib': mpl_version,
     }
 
     if not sys.platform == 'darwin':                    # To avoid a crash with our Mac app
         versions['system'] = platform.system()          # Linux, Windows, ...
     else:
         versions['system'] = 'Darwin'
 
-    try:
-        from matplotlib import __version__ as mpl_version
-        versions['matplotlib'] = mpl_version
-    except ImportError:
-        pass
-
-    try:
-        from numpy import __version__ as np_version
-        versions['numpy'] = np_version
-    except ImportError:
-        pass
-
-    try:
-        from pandas import __version__ as pd_version
-        versions['pandas'] = pd_version
-    except ImportError:
-        pass
-
-    try:
-        from larray import __version__ as larray_version
-        versions['larray'] = larray_version
-    except ImportError:
-        pass
+    def add_package_version(package_name):
+        try:
+            package = import_module(package_name)
+            versions[package_name] = package.__version__
+        except ImportError:
+            pass
+
+    add_package_version('numpy')
```

Comment:
no matplotlib anymore? isn't it supposed to be an optional dependency?

Reply:
At the beginning of `editor.py`, we have:
```python
import os
import re
import matplotlib
import numpy as np
```
But that may be considered as something to be changed.

Reply:
But I agree to have the most generic `get_versions` function (I only consider Qt as not optional).

Reply:
Hmm, sorry I forgot we made it a required dependency of the editor a while back.